spine-c/include/spine/IkConstraint.h

```diff
@@ -51,14 +51,17 @@ typedef struct spIkConstraint {
 	int bendDirection;
 	float mix;
 
+	int level;
```

Comment:
Could be internal, though it's not a big deal for this to appear in the public API. Same with Bone#sorted.